src/main/java/com/ashkan/moviesapi/entities/Actor.java

```diff
@@ -11,7 +11,7 @@
     private Collection<ActorsInMovies> actorsInMoviesById;
```

Comment:
I can't add this comment in the proper line, please take this into account as if it is in line 48.
In this type of relation **@OneToMany**, don't we need to use the **_cascade = CascadeType.ALL_** property? Please review this to understand if it is necessary or which type of cascade is necessary.

Reply:
done.